core/data/src/main/java/com/boostcamp/dreamteam/dreamdiary/core/data/repository/DefaultDreamDiaryRepository.kt

```diff
@@ -77,6 +53,23 @@ internal class DefaultDreamDiaryRepository @Inject constructor(
         )
     }
 
+    override suspend fun updateDreamDiary(
+        diaryId: String,
+        title: String,
+        diaryContents: List<DiaryContent>,
+        labels: List<String>,
+        sleepStartAt: Instant?,
+        sleepEndAt: Instant?,
+    ) {
+        dreamDiaryDao.updateDreamDiary(
+            diaryId = diaryId,
+            title = title,
+            body = makeBody(diaryContents = diaryContents),
+            sleepStartAt = sleepStartAt ?: Instant.now(),
+            sleepEndAt = sleepEndAt ?: Instant.now(),
```

Comment:
nullable하게 사용하면 안될 것 같습니다.


```suggestion
        sleepStartAt: Instant,
        sleepEndAt: Instant,
    ) {
        dreamDiaryDao.updateDreamDiary(
            diaryId = diaryId,
            title = title,
            body = makeBody(diaryContents = diaryContents),
            sleepStartAt = sleepStartAt,
            sleepEndAt = sleepEndAt,
```